### PR TITLE
Added missing constructor of Configuration in the Extension class

### LINKED ISSUE #1511: Unable to dump configuration via console config:dump
```bash
$ sf config:dump fos_rest

[LogicException]
No extensions with configuration available for "fos_rest"
```

### PR DESCRIPTION
**Pull request by hshn**
fixes #1511

### CODE REVIEW

**Comment by xabbuh** [comment]
@hshn Thanks for fixing this. But can you please revert the CS changes (we updated the StyleCI config recently so this shouldn't be necessary anymore)?

**Comment by hshn** [comment]
@xabbuh I've removed the CS changes, but the StyleCI configuration updates seems to be unavailable at branch 2.0 for now. What should I do?

**Comment by xabbuh** [comment]
@hshn Thank you! I'll merge the `1.7` branch up to `1.8`, `2.0`, and `master` tonight. After that this should work.

**Comment by xabbuh** [comment]
@hshn I have merged things up. Can you rebase here and trigger the hooks again?

**Comment by hshn** [comment]
@xabbuh rebased :)

**Comment by xabbuh** [comment]
:+1:

File: Tests/DependencyInjection/FOSRestExtensionTest.php [hunk 1]
```diff
@@ -603,6 +603,13 @@ public static function getShowExceptionData()
         );
     }
 
+    public function testGetConfiguration()
+    {
+        $configuration = $this->extension->getConfiguration([], $this->container);
+
+        $this->assertInstanceOf('FOS\RestBundle\DependencyInjection\Configuration', $configuration);
```
Comment (Ener-Getick) [inline Tests/DependencyInjection/FOSRestExtensionTest.php]: The short syntax should be used here: ``Configuration::class``

**Comment by Ener-Getick** [comment]
except my small comment, LGTM :+1:

File: Tests/DependencyInjection/FOSRestExtensionTest.php (hunk 1 above)
Comment (xabbuh) [inline Tests/DependencyInjection/FOSRestExtensionTest.php]: Imo we should keep this as is as we need to merge this into the 1.8 branch.

Comment (Ener-Getick) [inline Tests/DependencyInjection/FOSRestExtensionTest.php]: @xabbuh indeed you're right. That's fine for me then.

**Comment by Ener-Getick** [comment]
Merged in https://github.com/FriendsOfSymfony/FOSRestBundle/commit/deb09148df3fc5ae79628b7956a97626213752f9. Thanks !

**Pull request closed without merging**